packages/zowe-explorer/src/trees/dataset/DatasetActions.ts

```diff
@@ -653,17 +645,10 @@ export class DatasetActions {
         }
         selectedNodes = selectedNodes.filter((val) => !childArray.includes(val as IZoweDatasetTreeNode));
 
-        if (includedSelection || !node) {
-            // Filter out sessions and information messages
-            nodes = selectedNodes.filter(
-                (selectedNode) => selectedNode.getParent() && !SharedContext.isSession(selectedNode) && !SharedContext.isInformation(selectedNode)
-            ) as IZoweDatasetTreeNode[];
-        } else {
-            if (node.getParent() && !SharedContext.isSession(node) && !SharedContext.isInformation(node)) {
-                nodes = [];
-                nodes.push(node);
-            }
-        }
+        // Filter out sessions and information messages
+        let nodes = selectedNodes.filter(
```

Comment:
(nitpick) `nodes` can be made `const` if lines 703-710 were refactored to return instead of setting `nodes` to an empty array. That said, I'm not expecting you to change existing code that's not entirely related to this fix, just thinking of ways to optimize where possible 🙂 

It is a bit odd that those aforementioned lines have a pattern of `await Gui.infoMessage(...).then(...)` - but this code is pre-existing, so not requesting changes 😋